setup.py

```diff
@@ -10,9 +10,9 @@
     license="GPLv3",
     keywords="",
     url="https://github.com/raphaeldussin/om4labs",
-    packages=["om4labs"],
+    packages=["om4labs", "om4labs/diags",
+              "om4labs/diags/annual_bias_1x1deg"],
```

Comment:
Should we list each script separately, or just rely on the super wrapper?

Reply:
for some obscure reason I could not get the diags to load properly without adding explicitly the different subdirectories. maybe an issue to revisit later